### PR TITLE
T24 - Chocolate Wrapper

### PR DESCRIPTION
**Pull request by Ajay07Metamation**

### CODE REVIEW

File: Training/Program.cs [hunk 1]
```diff
@@ -1,7 +1,41 @@
...
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿//Program.cs
+//Chocolate Wrappers
+//To find the number of chocolates one can buy with a given amount
+//Numbers of wrappers remaining after exchanging wrappers with chocolate and money remaining
+// Sample Input : 20 3 5      Sample Output : 7 2 2 
+
+///Chocolate Wrappers
```
Comment (MetaNaveen) [inline Training/Program.cs:7]: Triple slashes are used for XML comments.

File: Training/Program.cs [hunk 2]
```diff
@@ -1,7 +1,41 @@
...
+Write ("Enter Price of chocolate : ");
+int.TryParse (ReadLine (), out int price);
+Write ("Enter Number of Wrappers to be Exchanged for a chocolate :");
+int.TryParse (ReadLine (), out int wrappers);
+if (money > 0 && price > 0 && wrappers > 1) {
+   var result = GetChocolate (money, price, wrappers);
+   WriteLine ($"\nThe Number of chocolates is {result.Choc}\n" +
+                      $"The Remaining Money is {result.RemMoney}\n" +
+                      $"The Remaining wrappers is {result.RemWrappers}");
+} else
```
Comment (MetaNaveen) [inline Training/Program.cs:22]: Statement could be on the same line.

**Review by MetaNaveen — COMMENTED** [review]
Working fine. Good.

**Comment by MetaNaveen** [comment]
Just resolve the above and close the PR yourself.

File: Training/Program.cs (hunk 2 above)
Comment (Ajay07Metamation) [reply Training/Program.cs:20]: Changes Done and Closing the PR

**Pull request closed without merging**